homeassistant/components/unifi/sensor.py

```diff
@@ -352,6 +353,38 @@ class UnifiSensorEntityDescription(
         unique_id_fn=lambda hub, obj_id: f"password-{obj_id}",
         value_fn=lambda hub, obj: obj.x_passphrase,
     ),
+    UnifiSensorEntityDescription[Devices, Device](
+        key="Device CPU utilization",
+        device_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=PERCENTAGE,
+        api_handler_fn=lambda api: api.devices,
+        available_fn=async_device_available_fn,
+        device_info_fn=async_device_device_info_fn,
+        name_fn=lambda device: "CPU utilization",
+        object_fn=lambda api, obj_id: api.devices[obj_id],
+        supported_fn=lambda hub, obj_id: hub.api.devices[obj_id].raw.get(
+            "system-stats", False
+        ),
```

Comment:
maybe it would make sense to add a `has_system_stats` to a device inside the aiounifi package?

Reply:
For now just make it a method as lambda shouldn't be multiline and I think this suggestion doesnt get short enough either
```suggestion
        supported_fn=lambda hub, obj_id: "system-stats" in hub.api.devices[obj_id].raw,
```